crates/gpui/src/platform/windows/dispatcher.rs

```diff
@@ -1,71 +1,104 @@
 use std::{
-    sync::{
-        atomic::{AtomicIsize, Ordering},
-        Arc,
-    },
     thread::{current, ThreadId},
+    time::Duration,
 };
 
 use async_task::Runnable;
-use flume::Sender;
 use parking::Parker;
 use parking_lot::Mutex;
-use windows::Win32::{Foundation::*, System::Threading::*};
+use util::ResultExt;
+use windows::{
+    Foundation::TimeSpan,
+    System::{
+        DispatcherQueue, DispatcherQueueController, DispatcherQueueHandler,
+        Threading::{
+            ThreadPool, ThreadPoolTimer, TimerElapsedHandler, WorkItemHandler, WorkItemOptions,
+            WorkItemPriority,
+        },
+    },
+    Win32::System::WinRT::{
+        CreateDispatcherQueueController, DispatcherQueueOptions, DQTAT_COM_NONE,
+        DQTYPE_THREAD_CURRENT,
+    },
+};
 
 use crate::{PlatformDispatcher, TaskLabel};
 
+macro_rules! generate_func {
```

Comment:
Let's remove both of these macros entirely.

Reply:
Done.